pipeline/outputs/s3.md

```diff
@@ -282,6 +289,23 @@ Example:
 
 Then, the records will be stored into the MinIO server.
 
+## Usage for Parquet Compression
+
+For parquet compression, it needs to install [columnify](https://github.com/reproio/columnify) in the running system or container at runtime.
```

Comment:
I'd say, "users need to install"

Can we flesh this out into an example with the FLuent Bit image so folks can easily figure it out?

From reaing this, I don't know how to use parquet quickly

Reply:
We'll plan to add an instruction of columnify command into Fluent Bit image at least.

Reply:
Not sure where we should put the update for using columnify command but I found that a minimal example to add columnify command in a Docker container:

```docker
FROM golang:1.22-alpine as builder

ENV ROOT=/go/src/cmd
WORKDIR ${ROOT}

RUN apk update && \
  apk add git

RUN go install github.com/reproio/columnify/cmd/columnify@latest

# hadolint ignore=DL3006
FROM debian:bullseye-slim as production

COPY --from=builder /go/bin/columnify /usr/bin/columnify
```

Reply:
I tested both of 1.22-bullseye and 1.22-alpine images. When using alpine images, I got succeeded to obtain non-dynamic executables for columnify.

Reply:
I think either this or something like it needs to be in the docs, or linked to from the docs.

Reply:
I added an example for building columnify command inside containers here: https://github.com/fluent/fluent-bit-docs/pull/1380/commits/94838f393644cce15c6883bc63a064661b33496f